docs/api/apiv3/endpoints/work-packages.apib

```diff
@@ -7,6 +7,7 @@
 | addAttachment       | Attach a file to the WP                                                  | **Permission**: edit work package      |
 | addComment          | Post comment to WP                                                       | **Permission**: add work package notes |
 | addRelation         | Adds a relation to this work package.                                    | **Permission**: manage wp relations    |
+| additionalTabs      | List of additional accessible work package tabs. Extended by plugins.    | **Permission**: defined by plugins     |
```

Comment:
Is this API blueprint docu manually edited or generated? I manually changed it for now, but am happy for any hints on tools/processes you use to manage/validate this file.

Reply:
Unfortunately, the entire blueprint process is manual. As blueprint has more or less lost against swagger/OpenAPI, there is no working tools that help our process here and my goal would be to switch to an OpenAPI generated format at some point in time.

Reply:
It will print out warnings/error on the console but it is still possible to use the originally used renderer (aglio)

1) Install it via npm: `npm i -g aglio` (I believe)
2) Run it via aglio: `aglio -s -i docs/api/apiv3/endpoints/work-packages.apib`

This will only render this one page but might be good enough to give it a glance over.

Reply:
thanks! turns out that moving the into plugins removes the need to document the endpoint here 🎉